### ISSUE TITLE
still not running properly

### ISSUE THREAD

**Issue by immerzu** [yuliskov/SmartTube#3751]
Some YT videos (2024-07-23) just won't run. The app worked great for years, but recently the worm has gotten into it.

**Comment by Mrunknow3** [comment]
Error not gone too

**Comment by aitronssesin** [comment]
Yep same issue here

**Comment by stamatis87** [comment]
22.65 works fine on my Sony Android TV but on the Shield Pro it's coming up with "Unknown source error" Response code 403. But if I leave it'll play after a couple of attempts.

**Comment by Patraputt** [comment]
Everyone, please update to 22.65. It'll work like a charm!

**Comment by immerzu** [comment]
> Everyone, please update to 22.65. It'll work like a charm!

Yes, it seems to be the case.

**Comment by SandeshBhat15** [comment]
not working on firetv stick videos are always loading

**Comment by Patraputt** [comment]
> not working on firetv stick videos are always loading

I've tested 22.65 on FireTv stick 4k Max and Chromecast with Google TV 4k, and Android TV, it's working well.

**Comment by Katerle** [comment]
FireTV 4K, logged in as User, 22.65 Beta does the yob like a charm. Thank you to all developers!

**Comment by notamaiar** [comment]
> Everyone, please update to 22.65. It'll work like a charm!

Unfortunately 22.65 stopped working for me this morning (403 error) after working fine last night. Had to revert to 22.38. ☹️

**Comment by hiroober** [comment]
Yes same here. 22.65 worked fine last night but stopped working this morning.

**Comment by 6jarjar6** [comment]
> Yes same here. 22.65 worked fine last night but stopped working this morning.

Same here worked, but now getting 403

**Comment by scaarup** [comment]
Is Youtube/Google trying to block this app?

**Comment by Katerle** [comment]
I think it seems so, they try blocking (all?) 3rd party apps...

**Comment by Grumpy-Githubr** [comment]
Nvidia Sheild Pro
Videos wouldn’t play this morning
Remembers to check for an update, save ver. 22.66 was available.
Updated to 22.26 and it;s working again.

**Comment by Katerle** [comment]
FireTV 4K, user logged in:
22.65 Beta: error 403
22.66 Beta: error 403
22.45 Stable: nothing works
22.65 Stable: no error

Seems YT is playing games...

**Comment by Patraputt** [comment]
22.65 not working any more. 22.67 makes everything stopped.

**Comment by Patraputt** [comment]
I don't like the 69 pose although I update to .69 and ghost playback returned back. 🙄
